utils/build-script-impl

```diff
@@ -2516,6 +2520,40 @@ for host in "${ALL_HOSTS[@]}"; do
                     LIBICU_BUILD_ARGS=()
                 fi
 
+                if [[ ! "${SKIP_BUILD_LIBXML2}" ]]; then
+                    BASE_INSTALL_DIR="$(get_host_install_destdir ${host})"
+                    LIBXML2_HEADERS="${BASE_INSTALL_DIR}/usr/include/libxml2"
+                    LIBXML2_BUILD_ARGS=(-DLIBXML2_INCLUDE_DIR:PATH="$LIBXML2_HEADERS")
+                    # Find static library
+                    LIBXML2_LIB=$(find "${BASE_INSTALL_DIR}"  -name 'libxml2.a')
+                    if [[ ! "${LIBXML2_LIB}" ]]; then
+                        LIBXML2_LIB=$(find "${BASE_INSTALL_DIR}" -name 'libxml2.so' -o -name 'libxml2.dll')
+                        LIBXML2_BUILD_ARG+=($LIBXML2_LIB)
+                    else
+                        LIBXML2_BUILD_ARG+=(-DLIBXML2_DEFINITIONS="-DLIBXML_STATIC")
+                    fi
+                    LIBXML2_BUILD_ARGS+=(-DLIBXML2_LIBRARY:PATH="$LIBXML2_LIB")
+                else
+                    LIBXML2_BUILD_ARGS=()
+                fi
+
+                if [[ ! "${SKIP_BUILD_ZLIB}" ]]; then
+                    BASE_INSTALL_DIR="$(get_host_install_destdir ${host})"
+                    ZLIB_LIB=$(find $BASE_INSTALL_DIR -name 'libz.a' -o \
+                               -name 'libz.so' -o -name 'libz.dll' | head -n 1)
```

Comment:
`find` wont work for windows, `libz.a` is the wrong name (`zlibstatic.lib` is the library on Windows).  I think that we should do the same as windows here - always statically link zlib.

Reply:
Done